docs/resources/elb_security_policy.md

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Dedicated Load Balance (Dedicated ELB)"
+---
+
+# huaweicloud_elb_security_policy
+
+Manages an ELB security policy resource within HuaweiCloud.
+
+## Example Usage
+
+```HCL
+resource "huaweicloud_elb_security_policy" "test" {
+name                  = "security_policy_test"
+description           = "this is a security policy"
+enterprise_project_id = "4579f2f5-cbe8-425a-8f32-53dcb9d9053a"
```

Comment:
suggest to put the `enterprise_project_id` at the end of this resource or delete it.

Reply:
done